modules/abstract-eth/src/abstractEthLikeCoin.ts

```diff
@@ -221,4 +222,11 @@ export abstract class AbstractEthLikeCoin extends BaseCoin {
    * @return a new transaction builder
    */
   protected abstract getTransactionBuilder(): TransactionBuilder;
+
+  /**
+   * @inheritdoc
+   */
+  getHashFunction(): Hash {
+    return createKeccakHash('keccak256');
+  }
```

Comment:
are we using this for any coin apart from Atom?
if not any we throw not implemented and get rid of `keccak`